_observing-your-data/query-insights/query-metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Query metrics
+parent: Query insights
+nav_order: 20
+---
+
+# Query metrics
+
+Query metrics involves comprehensive instrumentation along the search path to capture key [metrics](#metrics), such as aggregation types, query types, latency, and resource usage per query type. Open Telemetry (oTel) is the instrumentation framework. The telemetry data can be consumed using oTel metrics [exporters]({{site.url}}{{site.baseurl}}/observing-your-data/trace/distributed-tracing/#exporters).
```

Comment:
```suggestion
Query metrics involves comprehensive instrumentation along the search path to capture key [metrics](#metrics), such as aggregation types, query types, latency, and resource usage per query type. OpenTelemetry (OTel) is the instrumentation framework. The telemetry data can be consumed using OTel metrics [exporters]({{site.url}}{{site.baseurl}}/observing-your-data/trace/distributed-tracing/#exporters).
```

Reply:
Instrumentation refers to the tools and processes used to collect, measure, and report data about the performance and behavior of a system. How about "Query metrics _uses_ comprehensive instrumentation along the search path to capture key [metrics](#metrics), such as aggregation types, query types, latency, and resource usage per query type."?